decred/decred/dcr/vsp.py

```diff
@@ -305,22 +306,23 @@ def authorize(self, address):
             address (string): The base58-encoded pubkey address that the wallet
                 uses to vote.
         """
-        # An error is returned if the address is already set
-        # {'status': 'error', 'code': 6,
-        #     'message': 'address error - address already submitted'}
-        # First try to get the purchase info directly.
+        # An error is returned if the address isn't yet set
+        # {'status': 'error', 'code': 9,
+        #  'message': 'no address submitted',
+        # 'data': None}
         try:
             self.getPurchaseInfo()
             self.validate(address)
         except Exception as e:
             # code 9 is address not set
-            alreadyRegistered = (
+            addressNotSet = (
                 isinstance(self.err, dict)
                 and "code" in self.err
                 and self.err["code"] == 9
             )
-            if not alreadyRegistered:
+            if not addressNotSet:
                 raise e
+
```

Comment:
I think that the wording was wrong here. getPurchaseInfo will fail if we have not yet submitted and address. Submitting an address will fail if we have a code 6, `address already submitted`, but I don't think we can reach that, because we were just told that we have not submitted an address.